on musl libc off_t is always 64bit

This is needed for interception to build on 32 bit musl

--- a/projects/compiler-rt/lib/sanitizer_common/sanitizer_internal_defs.h
+++ b/projects/compiler-rt/lib/sanitizer_common/sanitizer_internal_defs.h
@@ -87,7 +87,8 @@
 // like pread and mmap, as opposed to pread64 and mmap64.
 // FreeBSD, Mac and Linux/x86-64 are special.
 #if SANITIZER_FREEBSD || SANITIZER_MAC || \
-  (SANITIZER_LINUX && defined(__x86_64__))
+  (SANITIZER_LINUX && \
+    (defined(__x86_64__) || (!defined(__GLIBC__) && !SANITIZER_ANDROID)))
 typedef u64 OFF_T;
 #else
 typedef uptr OFF_T;
